Backpack-SwiftUI/Price/Classes/BPKPrice.swift

```diff
@@ -0,0 +1,121 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2023 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_Common
+
+public struct BPKPrice: View {
+    public enum Size {
+        case large, small
+    }
+    
+    public enum Alignment {
+        case left, right
+    }
+    
+    @Binding public var price: String
+    @Binding public var leadingText: String?
+    @Binding public var lineThroughText: String?
+    @Binding public var trailingText: String?
+    @Binding public var alignment: Alignment
+    public let size: Size
+    
+    public init(
+        price: Binding<String>,
+        leadingText: Binding<String?> = .constant(nil),
+        lineThroughText: Binding<String?> = .constant(nil),
+        trailingText: Binding<String?> = .constant(nil),
+        alignment: Binding<Alignment> = .constant(.left),
```

Comment:
it seems you're not supposed to change this values inside this view, all this parameters can be just `let` without being bindings!